products/rds-mysql.md

```diff
@@ -0,0 +1,66 @@
+---
+title: RDS mysql
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: >-
+  https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+releaseImage: https://docs.aws.amazon.com/assets/r/images/aws_logo_dark.png
+activeSupportColumn: Active Support
+releases:
+  - releaseCycle: "8.0.32"
+    releaseDate: 2023-02-07
+    eol: 2024-02-01
+  - releaseCycle: "8.0.31"
+    releaseDate: 2022-11-10
+    eol: 2023-11-01
+  - releaseCycle: "8.0.30"
+    releaseDate: 2022-09-09
+    eol: 2023-09-01
+  - releaseCycle: "8.0.28"
+    releaseDate: 2022-03-11
+    eol: 2023-06-01
+  - releaseCycle: "8.0.27"
+    releaseDate: 2022-01-10
+    eol: 2023-04-20
+  - releaseCycle: "8.0.26"
+    releaseDate: 2021-10-25
+    eol: 2023-04-20
+  - releaseCycle: "8.0.25"
+    releaseDate: 2021-07-09
+    eol: 2023-04-20
+  - releaseCycle: "8.0.23"
+    releaseDate: 2021-03-31
+    eol: 2023-04-20
+  - releaseCycle: "5.7.41"
+    releaseDate: 2023-02-07
+    eol: 2023-10-01
+  - releaseCycle: "5.7.40"
+    releaseDate: 2022-11-11
+    eol: 2023-10-01
+  - releaseCycle: "5.7.39"
+    releaseDate: 2022-09-29
+    eol: 2023-09-01
+  - releaseCycle: "5.7.38"
+    releaseDate: 2022-06-06
+    eol: 2023-06-01
+  - releaseCycle: "5.7.37"
+    releaseDate: 2022-03-11
+    eol: 2023-06-01
+  - releaseCycle: "5.7.36"
+    releaseDate: 2022-01-10
+    eol: 2023-04-20
+  - releaseCycle: "5.7.34"
+    releaseDate: 2021-07-09
+    eol: 2023-04-20
+  - releaseCycle: "5.7.33"
+    releaseDate: 2021-03-31
+    eol: 2023-04-20
+
+---
+
+> AWS RDS mysql is an AWS implementation of the popular mysql database.
```

Comment:
Could you elaborate a little more ? For example :

> Amazon RDS for MySQL is a PaaS offering from Amazon for creating MySQL Databases on AWS. It makes it easier to set up, operate, and scale MySQL deployments on AWS cloud. It runs MySQL Community Edition...

Reply:
Ok I've replaced my the description and am open to further review on this.

Reply:
I rephrase a bit, but LGTM.